.github/workflows/composition-ci.yaml

```
@@ -50,3 +60,26 @@ jobs:
             composition/coverage/coverage-final.json
           retention-days: 14
           codecov-token: ${{ secrets.CODECOV_TOKEN }}
+
+      - name: Run linter on composition-go
+        uses: ./.github/actions/go-linter
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/go-linter": line 5: unexpected key "type" for definition of input "working-directory"

(action)